src/utils.ts

```diff
@@ -14,3 +14,11 @@ export function trimString(string?: string, length = 50) {
 export function stopPropagation(event: React.MouseEvent) {
     event.stopPropagation();
 }
+
+export function removeHttpsWwwPageParams(url?: string) {
+    if (!url) return '';
+    return url
+        .replace(/(^\w+:|^)\/\//, '')
+        .replace('www.', '')
+        .split('?')[0];
```

Comment:
Can we just use extractRootDomain from above? I know you probably wanted to keep the path, but making it clearer what the domain is is probably a good thing (think phishing…). If you do really want to keep it I'd still use the URL-constructor for this.